internal/slicer/slicer.go

```diff
@@ -183,6 +185,16 @@ func Run(options *RunOptions) (*Report, error) {
 					globbedPaths[extractInfo.Path] = append(globbedPaths[extractInfo.Path], relPath)
 					addKnownPath(relPath)
 				}
+
+				// Do not add paths with "until: mutate".
+				pathInfo, ok := pathInfos[extractInfo.Path]
```

Comment:
Isn't this pathInfo the same object that we've obtained above by calling slice.Contents[extractInfo.Path], and making sure it existed?  Can't we just save that one instead and drop the additional logic here?

Reply:
You are right!! Changed it to remove the unnecessary lookup.